### DISCUSSION TITLE
Prowler Doubt

### DISCUSSION THREAD

**Discussion by MANAVGK2038** [prowler-cloud/prowler#3229] [Q&A]
What exactly does PASS and FAIL status mean? Is it necessary to check the entries that have severity of medium/high/critical if the status is PASS?

At the moment can we scan multiple AWS Accounts at once?

**Comment by jfagoagas** [comment]
Hi @MANAVGK2038, a `PASS` status determines that the resource audited is properly configured against the check otherwise a `FAIL` will be raised. Regarding `PASS` is not strictly necessary to check `medium/high/critical` findings but it is a good way also to know what is properly configured, security and operational wise.

You can scan multiple AWS Accounts at once, to do it please follow this documentation https://docs.prowler.cloud/en/latest/tutorials/parallel-execution/

Thanks for using Prowler 🚀
